journal/modernized/26-chapter-23.adoc

```diff
@@ -126,7 +126,8 @@ to turn men from darkness to light, and from the power of Satan unto God.
 That Christ was the Light, the true Light,
 that enlightens every man that comes into the world;
 unto whom all mankind must be turned, and must come, if they are saved,
-etc. etc. It was a highly favoured, solemn meeting,
+etc. It was a highly favoured, solemn meeting,
+//This was a tough call to remove an "etc." I think it can make sense either way
```

Comment:
Haha, I thought I removed that comment so I could make it here. I don't know what to do with that etc., etc. Sometimes he uses only one, so I'm wondering if he just used two for emphasis? Seems fine either way to me, but for continuity, I removed one.

Reply:
Oh gosh, I'm not sure what to do here. I googled it a little, but it's hard to say. It sort of seems like originally there was a subtle distinction between using it once, and using it twice, and I'm guessing Wheeler had his reasons for doing it twice.

I searched the original, and it looks like (as far as I can tell) it only appears 3 times in the book (see image below).

My gut is that most modern readers would think this was an error, so I think I would vote for removing the second one, _but only in the *modernized* edition._ And then, of course, leaving both in the _original_. But I'm curious if @Henderjay (jason) agrees...

![2019-03-04_09-19-46](https://user-images.githubusercontent.com/7050938/53739170-1d8b6280-3e5f-11e9-9601-23854991c4e6.png)